docker/env/get_docker_repo.sh

```diff
@@ -0,0 +1,14 @@
+# Get the correct docker repo from ECR per region
```

Comment:
Here should be shebang line, I think : )

Reply:
fixed